procedures/auxtel/reduction_procedures/Parabolic_Focus_Analysis.ipynb

```diff
@@ -0,0 +1,584 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    location = "tucson"
</pre><p>There is a way to get this from an environment variable I'm pretty sure. I need to dig up how though.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Yes, I have done this previously with </p><p><em>os.environ["LSST_DDS_PARTITION_PREFIX"]</em></p>

Reply:
<p>Already fixed. Thanks!</p>